### PR TITLE
set application cookie

### PR DESCRIPTION
**Pull request by SrinivasanTarget**
https://github.com/getgauge/taiko/issues/349

### CODE REVIEW

File: lib/taiko.js [hunk 1]
```diff
@@ -381,6 +381,25 @@ module.exports.title = async () => {
...
+ * @example
+ * setCookie("CSRFToken","csrfToken", {url: "http://the-internet.herokuapp.com"})
+ * 
+ * @param {string} name - Cookie name.
+ * @param {string} value - Cookie value.
+ * @returns {Promise<Object>} - Object with the description of the action performed.
+ */
+module.exports.setCookie = async (name, value, options = {}) => {
+    validate();
+    if (options.url === undefined && options.domain === undefined) throw new Error('Atleast URL or Domain needs to be specified for setting cookies');
```
Comment (zabil) [inline lib/taiko.js]: > Atleast URL or Domain needs to be specified for setting cookies

can this be

Please specify a domain or URL for setting cookies

Comment (SrinivasanTarget) [reply lib/taiko.js]: Done

File: lib/taiko.js [hunk 2]
```diff
@@ -381,6 +381,25 @@ module.exports.title = async () => {
     return result.result.value;
 };
 
+/**
+ * Sets a cookie with the given cookie data; may overwrite equivalent cookie if they exist.
+ *
+ * @example
+ * setCookie("CSRFToken","csrfToken", {url: "http://the-internet.herokuapp.com"})
```
Comment (nehashri) [inline lib/taiko.js]: Can you please add an example with domain too.

Comment (SrinivasanTarget) [reply lib/taiko.js]: Done

File: lib/taiko.js [hunk 3]
```diff
@@ -381,6 +381,27 @@ module.exports.title = async () => {
...
+ * setCookie("CSRFToken","csrfToken", {domain: "appium.io"})
+ * 
+ * @param {string} name - Cookie name.
+ * @param {string} value - Cookie value.
+ * @param {Object} options.
```
Comment (nehashri) [inline lib/taiko.js]: Description of the options param would be useful.

**Comment by SrinivasanTarget** [comment]
@NivedhaSenthil ping

**Pull request merged**